Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -25,6 +25,7 @@
 from acts.examples.odd import getOpenDataDetector
 
 ttbar_pu200 = False
+g4_simulation = False
```

Comment:
should we also adopt the itk full chain?

Reply:
I am pretty sure the ITk doesn't work with G4 (at least it didn't use to). But If I am wrong I would be more than happy to had it there